webserver/webserver/src/main/java/io/helidon/webserver/NettyWebServer.java

```diff
@@ -171,6 +162,9 @@
 
             bootstraps.put(name, bootstrap);
         }
+
+        // Log entry that also initializes NettyInitializer
+        LOGGER.fine(() -> NettyInitializer.getMaxOrderProperty() + " set to " + NettyInitializer.getMaxOrderValue());
```

Comment:
Please don't do that. 
JVM loads classes when bytecode is reached (sometimes eager but this depends)

Be explicit and not depending on "log level"